pubspec.yaml

```diff
@@ -1,11 +1,11 @@
 name: flutter_web_auth
 description: Flutter plugin for authenticating a user with a web service.
-version: 0.2.4
+version: 1.0.0
```

Comment:
Please don't update the version, I'll do that when I publish the next version ☺️

Reply:
Sure 👍